RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -101,79 +100,96 @@ boolean warAdventure()
 
 boolean L12_getOutfit()
 {
-	if (internalQuestStatus("questL12War") < 0 || internalQuestStatus("questL12War") > 0)
+	if (internalQuestStatus("questL12War") != 0)
 	{
 		return false;
 	}
 
+	// noncombat when adventuring at The Hippy Camp (Verge of War)
+	set_property("choiceAdventure139", "3");	//fight a War Hippy (space) cadet for outfit pieces
+	set_property("choiceAdventure140", "3");	//fight a War Hippy drill sergeant for outfit pieces
+	set_property("choiceAdventure141", "1");	//if wearing [Frat Boy Ensemble] get 50 mysticality
+	set_property("choiceAdventure142", "3");	//if wearing [Frat Warrior Fatigues] start the war or skip adventure
+	
+	// noncombat when adventuring at Orcish Frat House (Verge of War)
+	set_property("choiceAdventure143", "3");	//fight a War Pledgefor outfit pieces
+	set_property("choiceAdventure144", "3");	//fight a Frat Warrior drill sergeant for outfit pieces
+	set_property("choiceAdventure145", "1");	//if wearing [Filthy Hippy Disguise] get 50 muscle
+	set_property("choiceAdventure146", "3");	//if wearing [War Hippy Fatigues] start the war or skip adventure
+
+	// if you already have the war outfit we don't need to do anything now
 	if (haveWarOutfit())
 	{
 		return false;
 	}
-
-	set_property("choiceAdventure143", "3");
-	set_property("choiceAdventure144", "3");
-	set_property("choiceAdventure145", "1");
-	set_property("choiceAdventure146", "1");
-
-	if((get_property("auto_orcishfratboyspy") == "done") && !in_hardcore())
+	
+	//heavy rains softcore pull handling
+	if(!in_hardcore() && (auto_my_path() == "Heavy Rains"))
 	{
-		pullXWhenHaveY($item[Beer Helmet], 1, 0);
-		pullXWhenHaveY($item[Bejeweled Pledge Pin], 1, 0);
-		pullXWhenHaveY($item[Distressed Denim Pants], 1, 0);
+		//TODO add hippies support here and in heavy rains file for copying hippy spy instead.
+		if(get_property("auto_hippyInstead").to_boolean())
+		{
+			pullXWhenHaveY($item[Reinforced Beaded Headband], 1, 0);
+			pullXWhenHaveY($item[Round Purple Sunglasses], 1, 0);
+			pullXWhenHaveY($item[Bullet-proof Corduroys], 1, 0);			
+		}
+		// auto_orcishfratboyspy indicates that rainman was already used to copy an orcish frat boy in heavy rains. if it failed to YR pull missing items
+		if((get_property("auto_orcishfratboyspy") == "done") && !get_property("auto_hippyInstead").to_boolean())

```

Comment:
Should probably duplicate the `auto_orcishfratboyspy` logic for the appropriate hippy to yellow ray in heavy rains

Reply:
I was planning on doing this in a future update

Reply:
feature added on commit 8169349